### DISCUSSION TITLE
RTL SDR, DAB FM local radio source

### DISCUSSION THREAD

**Discussion by Shadoweee77** [music-assistant/support#2092] [Feature requests and ideas]
Hey guys,

first of all thanks for Your awesome work, MA is great!

So I've got a problem that fits MA but isn't restricted to it - as some of You know there are those DVBT receivers (commonly known as RTL SDR), that one could use to receive DAB and FM radio and stream to network - it would be Your local source.

I've searched on this topic quite a lot, but haven't found any solutions really - do any of You know anything like that? What are Your opinions on this idea / implementing it as MA source?

Why do it locally instead of using internet radio? Internet outages - quite common in my area and the pure joy/satisfaction of having things done from A to Z locally.

**Comment by OzGav** [comment]
Please don’t cross post between discord and here. I have responded on discord

**Comment by marcelveldt** [comment]
The most easy solution for this if you wrap up some scripts to publish the local DVB (or any other local source, like AUX-in or whatever) and publish it to a stream URL. Then add that stream URL manually as radio station and voila. You created your own stream.

A more advanced take on this would be creating a music provider for that DVB tuner but that is a lot more work as it involves scanning for stations etc. etc.
